tests/lib/rules/require-ember-lifeline.js

```diff
@@ -1,26 +1,31 @@
 const rule = require('../../../lib/rules/require-ember-lifeline');
-const MESSAGE = rule.meta.message;
+const getMessage = rule.meta.message;
 const RuleTester = require('eslint').RuleTester;
 const ruleTester = new RuleTester();
```

Comment:
We *can*, but do we care? If we do, we should do this across all the test files. That should be done separately in a different PR. Thoughts?